### DISCUSSION TITLE
Data sync/beatmap (+play history) exporting

### DISCUSSION THREAD

**Discussion by RadiatedExodus** [ppy/osu#19477] [Ideas] [answered]
Right now trying to export beatmap data and play history for respective beatmaps require you to literally copy your entire game data folder (AppData on Windows, Android/data on Android which is locked since Android 11 and you have to use SAF to access it), it would be better if there was data syncing between multiple devices under a single account (syncing play time, downloaded beatmaps, etc but mainly those 2)

If theres no plans to do that then could there be atleast a button to export beatmaps (and play history if possible) with a button instead?

**Comment by peppy** [comment, marked as answer]
If you are looking for anything not already on your online profile, please open a request against `osu-web`. We don't have any plan on having any data sync that is separate from the online profile system (to keep everything in one place).
